R/helpers.R

```diff
@@ -215,3 +215,54 @@ create_dummy_parent_package <- function(mn, children) {
   list(parent = create_response,
        children = children)
 }
+
+
+#' Create dummy attributes data frame
+#'
+#' @param numberAttributes (integer) Number of attributes to be created in the table
+#' @param factors (character) Optional vector of factor names to include.
+#'
+#' @return dataframe
+#' @export
+#'
+#' @examples
+create_dummy_attributes_dataframe <- function(numberAttributes, factors=NULL) {
+  names <- sapply(1:numberAttributes, function(x){ paste0("Attribute ", x) })
+  
+  if(!is.null(factors)) {
+    domains <- c(rep("textDomain", numberAttributes - length(factors)),
+                 rep("enumeratedDomain", length(factors)))
+    names[(numberAttributes - length(factors) + 1):numberAttributes] <- factors
+  }
+  
+  df <- data.frame(attributeName = names,
```

Comment:
I try to avoid using `df` as a variable name since `df` is an in-built function and this masks that function from the rest of the body of your function. A more descriptive name that is not already in use would be great.